docs/microbit_micropython_api.rst

```diff
@@ -266,6 +332,25 @@ There is an I2C bus on the micro:bit that is exposed via the `i2c` object.  It h
     # write buf to device with addr; repeat=True means a stop bit won't be sent.
     i2c.write(addr, buf, repeat=False)
 
+Sound **V2**
+------------
+
+A set of expressive sounds are available to the micro:bit **V2**. They can be
+accessed via the ``microbit`` module and played with the ``audio`` module.
```

Comment:
Can we link to the audio module here?

Reply:
Done in https://github.com/bbcmicrobit/micropython/pull/692/commits/8fe52d83b6eadf2dd74d94c71f4dbbc27c0ed779, nit sure whether to link to `microbit` module too, but perhaps in this context, the link to audio is most important